docs/http.md

```diff
@@ -0,0 +1,123 @@
+# HTTP on Spark
+### A library for interacting with HTTP services from Spark
+
+- **Flexible**: Encodes the entire HTTP protocol in spark for full control of web requests
```

Comment:
s/spark/Apache Spark/

Reply:
Sorry I dont quite understand what you mean here

Reply:
Sorry, replace spark with Apache Spark: https://en.wikipedia.org/wiki/Sed#Substitution_command

Reply:
Done